5calls/app/src/main/java/org/a5calls/android/a5calls/controller/SettingsActivity.java

```diff
@@ -225,6 +223,8 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
                 } else {
                     AccountManager.Instance.setUserName(getActivity(), null);
                 }
+            } else if (TextUtils.equals(key, "prefsKeyScriptTextSize")) {
+                AccountManager.Instance.setScriptTextSize(getActivity(), sharedPreferences.getString(AccountManager.KEY_SCRIPT_TEXT_SIZE, "16"));
```

Comment:
Suggest converting to float before sending to AccountManager to make it consistent type with the getter.